mapswipe_workers/mapswipe_workers/utils/slack_helper.py

```diff
@@ -0,0 +1,34 @@
+"""Initialize slack client with values provided by the config file"""
+
+import json
+import slack
+
+from mapswipe_workers.definitions import CONFIG_PATH
+
+
+def send_slack_message(message_type: str, project_name: str, project_id: str = None):
+
+    with open(CONFIG_PATH) as config_file:
+        config = json.load(config_file)
+        slack_channel = config["slack"]["channel"]
+        slack_client = slack.WebClient(token=config["slack"]["token"])
+
+    if message_type == "success":
+        message = (
+            "### PROJECT CREATION SUCCESSFUL ###\n"
+            + "Project Name: {0}\n".format(project_name)
+            + "Project Id: {0}\n\n".format(project_id)
+            + "Make sure to activate the project using the manager dashboard.\n"
+            + "Happy Swiping. :)"
+        )
+    elif message_type == "fail":
+        message = (
+            "### PROJECT CREATION FAILED ###\n"
+            + "Project Name: {0}\n".format(project_name)
+            + "Project draft is deleted."
+        )
+    else:
+        # TODO: Raise an Exception
+        pass
+
+    slack_client.chat_postMessage(channel=slack_channel, text=message)
```

Comment:
just for clarification: we don't need the username anymore, right?

Reply:
As far as I understand the user parameter is only used for sending an ephemeral message: "Ephemeral messages are only visible to an assigned user in a specified channel, is nearly the same as sending a regular message, but with an additional user parameter." https://slack.dev/python-slackclient/basic_usage.html

So yes, we do not nee the username anymore